### ISSUE TITLE
Carthage Support

### ISSUE THREAD

**Issue by cikpis** [usabilla/usabilla-u4a-ios-swift-sdk#20]
Hi guys, I was trying to integrate your SDK through Carthage but even though I've installed all the dependencies I got the follow error:

```
dyld: Library not loaded: @rpath/PromiseKit.framework/PromiseKit
Referenced from: /Users/XXX/Library/Developer/CoreSimulator/Devices/2B989444-4973-4175-9A95-A620774FE2D5/data/Containers/Bundle/Application/92037B90-ACC2-4CB8-9BC1-1D5F96237066/XXX.app/Frameworks/UsabillaFeedbackForm.framework/UsabillaFeedbackForm
Reason: Incompatible library version: UsabillaFeedbackForm requires version 1.0.0 or later, but PromiseKit provides version 0.0.0
```

Any clue?

Also why are you guys not supporting Carthage?

**Comment by cikpis** [comment]
https://github.com/mxcl/PromiseKit/issues/620

Just wondering how with CocoaPods all is working :)

**Comment by grimabe** [comment]
Hi @cikpis 

Thank you for bringing up this issue.

I tried to setup a project with Carthage and I am facing the same issue.
I will look on my side if I find anything.

There is no reason to not support Carthage, we will add this to our list.

In any case it should be fixed in a future release by removing the dependency on PromiseKit as of #15 (which has been done on our side but is currently under testing/approval).

I'll let you know.

**Comment by cikpis** [comment]
Hi @grimabe, I close the issue due to the problem is with PromiseKit. Please follow this https://github.com/mxcl/PromiseKit/issues/620

If you want you can reopen it and handle internally with your team ;)

**Comment by cikpis** [comment]
Hi @grimabe, after fetching again PromiseKit with the problem fixed, I'm facing another issue:
```PromiseKit was compiled with optimization - stepping may behave oddly; variables may not be available.```

Here the screenshot:
<img width="1552" alt="screen shot 2017-01-27 at 12 39 47 pm" src="https://cloud.githubusercontent.com/assets/895478/22369877/ba9e14fa-e48e-11e6-8d5d-4cc5d94a699f.png">

Do you have nay cue about it? Also, when will be the version without PromiseKit public? When are you guys planning to support Carthage?

I have a call today with Wouter Koch at 3.00pm GMT time. Could you fill him with the answers?

Thanks in advance :)

**Comment by gPinato** [comment]
@cikpis We have just released v3.2.0 of the SDK.
This version has no dependencies and supports carthage.

Let me know if you encounter any problem with it.

**Comment by cikpis** [comment]
@gPinato cool! thanks! I'm gonna integrate it and I'll let you know.
